genesyscloud/user/resource_genesyscloud_user_utils.go

```diff
@@ -871,6 +897,36 @@ func getSdkUtilizationTypes() []string {
 	return types
 }
 
+func buildVoicemailUserpoliciesRequest(voicemailUserpolicies []interface{}) platformclientv2.Voicemailuserpolicy {
+	var request platformclientv2.Voicemailuserpolicy
+	if extractMap, ok := voicemailUserpolicies[0].(map[string]interface{}); ok {
+		alertTimeoutSeconds := extractMap["alert_timeout_seconds"].(int)
```

Comment:
If you omit just `alert_timeout_seconds` from `voicemail_userpolicies`, you'll see the error "alert timeout too small". This is probably because the zero value for integers is being stored in `alertTimeoutSeconds`, which is zero. If you change this to `alertTimeoutSeconds, ok := ...` , and then only set the value of  `AlertTimeoutSeconds` if ok is true, then that should fix the problem